priv/.gitignore

```diff
@@ -29,3 +29,11 @@ erl_crash.dump
 elixir-*.tar
 
 .elixir_ls/
+
+#generated applications and source
```

Comment:
Oh then - having them here - why we need to ignore these in the top-most `.gitignore` again?
Also I'd simplify it to just: `apps/`